python/ray/scripts/scripts.py

```diff
@@ -700,7 +732,9 @@ def stack():
 cli.add_command(submit)
 cli.add_command(teardown)
 cli.add_command(teardown, name="down")
+cli.add_command(kill, name="kill_node")
```

Comment:
Also we should guard this somehow so that users aren't doing this to kill clusters; maybe 
`name=_kill_random_node`?